instrumentation/opentelemetry_ecto/mix.exs

```diff
@@ -6,7 +6,7 @@ defmodule OpentelemetryEcto.MixProject do
       app: :opentelemetry_ecto,
       description: description(),
       version: "1.0.0",
-      elixir: "~> 1.10",
+      elixir: "~> 1.12",
```

Comment:
We want to support the oldest supported Elixir version we can, so please revert to 1.10.